ui/packages/catalog-ui-search/src/main/webapp/component/oauth-item/oauth-item.view.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import * as React from 'react'
+import OauthItem from './oauth-item'
+const Marionette = require('marionette')
+const user = require('../singletons/user-instance.js')
+
+module.exports = Marionette.ItemView.extend({
+  template() {
+    return <OauthItem model={this.model} />
+  },
+  removeModel() {
```

Comment:
How does this get called?  By an existing component?

Reply:
It's called when someone tries to remove all the notifications. I think it's called from `notification-group.view.js`